src/common/transfer-to-modal/index.tsx

```diff
@@ -58,6 +60,7 @@ const TransferToModal = ({ transferTo, open, onCancel }: TransferToModalProps) =
   const handleTransfer = () => {
     dispatch(setTransferTo(toAddress));
     onCancel();
+    trackEvent('Aggregator - Set transfer to');
```

Comment:
Ditto what I said on transfer position: would be cool to also know when it was opened, not only submitted :blus

Reply:
Same as before